camelot/view/action_steps/change_object.py

```diff
@@ -346,7 +346,7 @@ def __init__(self, objects, admin, validate=True):
         self.subtitle = _('Please review the data below.')
         self.icon = Icon('file-excel') # 'tango/32x32/mimetypes/x-office-spreadsheet.png'
         self.invalid_rows = set()
-        self.columns = admin.get_columns()
+        self.columns = [(field, admin.get_field_attributes(field)) for field in admin.get_columns()]
```

Comment:
deze komen ook terecht in de onetomany editor, dus mogen denk ik ook gewoon weg

Reply:
done